Kernel/FormulaVarIterator.cpp

```diff
@@ -180,13 +182,17 @@ bool FormulaVarIterator::hasNext()
 
               _instructions.push(FVI_BIND);
               _vars.push(sd->getVariables());
+              _instructions.push(FVI_TERM_LIST);
```

Comment:
You can remove this line and the next one, since the sort of an `SF_FORMULA` term must be` $o `and therefore cannot contain any variables.

Reply:
This is the `SF_LET` case, I think the GitHub code viewer tricked you!

Reply:
Quite right!